pkg/executor/importer/import.go

```diff
@@ -1431,24 +1431,6 @@ type JobImportResult struct {
 	Warnings []contextutil.SQLWarn
 }
 
-// GetMsgFromBRError get msg from BR error.
-// TODO: add GetMsg() to errors package to replace this function.
-// see TestGetMsgFromBRError for more details.
```

Comment:
(in my memory) I wrote `TestGetMsgFromBRError` to ensure the message prefix is stripped. Now we have `GetErrStackMsg`, we can delete that unit test